plugin/timeout/wrappers.gotpl

```diff
@@ -0,0 +1,38 @@
+{{ reserveImport "context"  }}
+{{ reserveImport "time"  }}
+
+{{ range $w := .Wrappers }}
+    type {{$w.Type}} struct {
+        cfg TimeoutsConfig
+        parent {{$w.ObjectType}}
+    }
+
+    func New{{$w.Type}}(parent {{$w.ObjectType}}, cfg TimeoutsConfig) *{{$w.Type}}{
```

Comment:
what about to generate a method of the original resolver type, which wraps the receiver?
```go
NewOriginalResolver(...).WithTimeouts(cfg)
```

Reply:
or `WithDecorators(...)` which will apply all available decorators

Reply:
added decorators generation